Forta-Bots-Challenge-4/liquidatable-positions/src/finding.ts

```diff
@@ -0,0 +1,38 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+import { Liquidatable, PositionData } from "./agent.utils";
+
+export const createNewPositionFinding = (baseAssetAddresses: string[], newPosition: PositionData) => {
+  const newPositionToken = newPosition.token.toLowerCase();
+  const newPositionOwner = newPosition.owner.toLowerCase();
+  const newPositionBaseAsset = newPositionToken === baseAssetAddresses[0].toLowerCase() ? "USDC" : "WETH";
```

Comment:
```suggestion
  const newPositionBaseAsset = newPositionToken === BASE_ASSET_USDC.toLowerCase() ? "USDC" : "WETH";
```
Could update file to import `BASE_ASSET_USDC` and use it in this comparison, then avoid having the `baseAssetAddresses` parameter.